api/include/mfxstructures.h

```diff
@@ -1968,14 +2101,17 @@ enum {
     MFX_VP9_SEGMENT_FEATURE_SKIP        = 0x0008 /* (0,0) MV, no residual */
 };
 
+MFX_PACK_BEGIN_X(4)
 typedef struct {
     mfxU16  FeatureEnabled;         /* see enum SegmentFeature */
     mfxI16  QIndexDelta;
     mfxI16  LoopFilterLevelDelta;
     mfxU16  ReferenceFrame;        /* see enum VP9ReferenceFrame */
     mfxU16  reserved[12];
 } mfxVP9SegmentParam;
+MFX_PACK_END()
 
+MFX_PACK_BEGIN()
```

Comment:
mfxU64          reserved1 - why not _MAX?

Reply:
fixed